src/components/notifications/notifications.tsx

```diff
@@ -21,6 +21,8 @@ export interface NotificationsProps {
     notifications: Observable<NotificationInfo>;
 }
 
+require('./notifications.scss');
```

Comment:
I can't push to your branch and GH can't comment on unchanged lines, so here's what I was suggesting.

Remove this CSS file and import, instead, on line 50/52 below of the `div`, add:
```tsx
               <div onClick={(e) => {
                 // ... elided ... don't change anything with the `onClick`
                }} style={{
                    // fit long words
                    overflowWrap: 'break-word',
                    maxWidth: '240px'
                }}>
```

Reply:
I totally agree, Thanks a lot, I will make the changes and let you know